projects/angular/src/forms/combobox/_combobox.clarity.scss

```
@@ -50,132 +34,59 @@
 
   // Focus indicator, this is a custom implementation compared with other controls due to DOM structure and potential growth of height
   .clr-focus-indicator {
-    @include mixins.css-var(
-      background-color,
-      clr-forms-focused-color,
-      forms-variables.$clr-forms-focused-color,
-      variables.$clr-use-custom-properties
-    );
-    height: variables.$clr_baselineRem_2px;
+    background-color: forms-variables.$clr-forms-focused-color;
+    height: tokens.$cds-global-space-2;
     width: 0;
     transition: width 0.2s ease;
     position: absolute;
-    bottom: -(variables.$clr_baselineRem_1px);
+    bottom: calc(-1 * #{tokens.$cds-global-space-1});
     left: 0;
-    @include mixins.css-var(
-      border-bottom-left-radius,
-      clr-combobox-border-radius,
-      combobox-variables.$clr-combobox-border-radius,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      border-bottom-right-radius,
-      clr-combobox-border-radius,
-      combobox-variables.$clr-combobox-border-radius,
-      variables.$clr-use-custom-properties
-    );
+    border-bottom-left-radius: combobox-variables.$clr-combobox-border-radius;
+    border-bottom-right-radius: combobox-variables.$clr-combobox-border-radius;
   }
   .clr-focus {
     width: 100%;
   }
   .clr-error .clr-focus-indicator {
-    @include mixins.css-var(
-      background-color,
-      clr-forms-invalid-color,
-      forms-variables.$clr-forms-invalid-color,
-      variables.$clr-use-custom-properties
-    );
+    background-color: forms-variables.$clr-forms-invalid-color;
   }
 
   .clr-combobox-wrapper {
     position: relative;
 
     //Dimensions
-    padding: 0 caret-gap() 0 combobox-variables.$clr-combobox-trigger-left-padding;
-    @include mixins.css-var(
-      padding-left,
-      clr-combobox-padding-left,
-      combobox-variables.$clr-combobox-trigger-left-padding,
-      variables.$clr-use-custom-properties
-    );
-
-    @include mixins.css-var(
-      min-height,
-      clr-combobox-min-height,
-      combobox-variables.$clr-combobox-min-height,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      min-width,
-      clr-combobox-min-width,
-      combobox-variables.$clr-combobox-min-width,
-      variables.$clr-use-custom-properties
-    );
+    padding: 0 caret-gap() 0 combobox-variables.$clr-combobox-trigger-padding-left;
```

We should keep the naming convention or use a deprecation clause in `_variables.combobox.scss`.

```suggestion
    padding: 0 caret-gap() 0 combobox-variables.$clr-combobox-trigger-left-padding;
```

---

Good catch! probably legacy override when the `padding` has been simpler.

---

Returning, for backward compatibility reasons.